TF_Service_dotNet/TouchFree/Configuration/VersionManager.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Reflection;
+
+namespace Ultraleap.TouchFree.Library.Configuration
+{
+    public static class VersionManager
+    {
+        public static string Version
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(_version))
+                {
+                    _version = Assembly.GetExecutingAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion;
+                }
+
+                return _version;
+            }
+        }
```

Comment:
Could replace with `_version ??= Assembly.GetExecutingAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>()!.InformationalVersion;`
Set `_version` default to null.
Note I added a null suppression operator to fix warning that the attribute could be missing.

Reply:
I've updated this to handle the attribute not existing instead